docs/docker/README.md

```diff
@@ -1,44 +1,69 @@
 # Minio Docker Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
 
-## Prerequisites
-Docker installed on your machine. Download the relevant installer from [here](https://www.docker.com/community-edition#/download).
+This quickstart guide describes how to quickly install and run a Minio Server Docker container.
 
-## Run Standalone Minio on Docker.
-Minio needs a persistent volume to store configuration and application data. However, for testing purposes, you can launch Minio by simply passing a directory (`/data` in the example below). This directory gets created in the container filesystem at the time of container start. But all the data is lost after container exits.
+1. [Install Docker](#installdocker) 
+2. [Run Standalone Minio on Docker](#runstandalong) 
+3. [Run Distributed Minio on Docker](#rundistributed) 
+
+## <a name="installdocker"></a>1. Install Docker
+Install Docker using these instructions: <https://www.docker.com/community-edition#/download>.
+
+## <a name="runstandalong"></a>2. Run Standalone Minio on Docker
```

Comment:
Anchor names should use - separated notation, e.g., install-docker.  Correct typo (standalong to standalone)